ios/DroidKaigi 2021/Debug.xcodeproj/project.pbxproj

```diff
@@ -144,6 +137,42 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		42B134C0F3C46E32AF4355A0 /* Make kotlin framework */ = {
```

Comment:
Please remove this phase:pray:

Reply:
I'm sorry, I didn't realize that🙇‍♂️ 8018ca6